pytest/tests/mocknet/mirror.py

```diff
@@ -451,6 +474,14 @@ def run_env_cmd(args, traffic_generator, nodes):
     parser.add_argument('--start-height', type=int)
     parser.add_argument('--unique-id', type=str)
     parser.add_argument('--local-test', action='store_true')
+    parser.add_argument('--type',
```

Comment:
"type" is ambiguous, maybe rename this to --host-type or --hosts, and --filter to --filter-hosts?